digdag-standards/src/main/java/io/digdag/standards/operator/jdbc/LockConflictException.java

```diff
@@ -0,0 +1,28 @@
+package io.digdag.standards.operator.jdbc;
+
+import java.sql.SQLException;
+
+public class LockConflictException
+        extends RuntimeException
```

Comment:
I think checked-exception (extending Exception instead of RuntimeException) works better for this use case because this exception must be caught and must not be re-thrown.